docs/src/getting_started.md

```diff
@@ -8,8 +8,8 @@ You can obtain the release build via the [download page](https://zed.dev/downloa
 
 ### Configure Zed
 
-Use `CMD + ,` to open your custom settings to set things like fonts, formatting settings, per-language settings and more. You can access the default configuration using the `Zed > Settings > Open Default Settings` menu item. See Configuring Zed for all available settings.
+Use `⌘` `,` to open your custom settings to set things like fonts, formatting settings, per-language settings and more. You can access the default configuration using the `Zed > Settings > Open Default Settings` menu item. See Configuring Zed for all available settings.
```

Comment:
I think it will look a little odd to have the two keys next to each other like this:

<img width="825" alt="Screenshot 2024-01-26 at 6 41 57 PM" src="https://github.com/zed-industries/zed/assets/1486634/4c110de7-7d57-4155-97b6-a91a9555b109">

What if we kept the plus, but just left it out of the inline code block?

`⌘` + `,`

<img width="775" alt="Screenshot 2024-01-26 at 6 42 35 PM" src="https://github.com/zed-industries/zed/assets/1486634/8c2b72eb-00d6-4120-8207-0af01a4367a1">

Reply:
That is equally clear to me. Though, `⌘` + `K`, `⌘` + `S` looks like two distinct steps and not one single shortcut.

Reply:
It's the way GitHub denotes their keyboard shortcuts, so it seems like a good pattern to follow, IMO:

<img width="712" alt="Screenshot 2024-01-26 at 6 50 55 PM" src="https://github.com/zed-industries/zed/assets/1486634/b4253e32-298f-48f9-9ca6-439a4fae2f17">